src/kixi/weymouth/poly.clj

```diff
@@ -0,0 +1,27 @@
+(ns kixi.weymouth.poly)
```

Comment:
I don't believe this namespace is used anywhere and it adds little value, it can be removed.